src/Section.tsx

```diff
@@ -106,14 +109,26 @@ const Section: React.FC<Props> = ({
         <div
           css={{
             display: "flex",
-            width: "120px",
+            width: "100px",
             flexDirection: "column",
             padding: "5px"
           }}
         >
-          {instructors.map(i => (
-            <div key={i}> {i} </div>
-          ))}
+            <div> {instructors.join(', ')} </div>
+         </div>
+         <div
+           css={{
+             display: "flex",
+             width: "80px",
+             flexDirection: "column",
+             padding: "5px"
+           }}
+         >
+         <div> {type} </div>
+         </div>
+         <div css={{ width: "20px", maxWidth: "50vw", padding: "5px" }}>
+           {" "}
+           {fixCredit(minUnits, maxUnits)}
```

Comment:
Thoughts on whether or not we should say like "4 credits"? I'm not sure the number alone is enough for people to get that it's credit amount

Reply:
Hmm, I think we can do that later. I think we should modify the page to make it look nicer and the boxes containing info to be bigger. Because right now, adding "4 credits" make it quite cramped. But we can definitely do it later on or I can work on it at a later time